5-AccessControl/2-call-api-groups/README.md

```diff
@@ -0,0 +1,634 @@
+   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `msal-react-spa`.
+   - Under **Supported account types**, select **Accounts in this organizational directory only**.
+   - In the **Redirect URI (optional)** section, select **Single-page application** in the combo-box and enter the following redirect URI: `http://localhost:3000/`.
+1. Select **Register** to create the application.
+1. In the app's registration screen, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
+1. Select **Save** to save your changes.
+1. In the app's registration screen, select the **API permissions** blade in the left to open the page where we add access to the APIs that your application needs.
+   - Select the **Add a permission** button and then:
+       - Ensure that the **Microsoft APIs** tab is selected.
+       - In the *Commonly used Microsoft APIs* section, select **Microsoft Graph**
+       - In the **Delegated permissions** section, select the **User.Read**, **GroupMember.Read.All** in the list. Use the search box if necessary.
+       - Select the **Add permissions** button at the bottom.
+   - **GroupMember.Read.All** requires admin to consent. Select the **Grant/revoke admin consent for {tenant}** button, and then select **Yes** when you are asked if you want to grant consent for the requested permissions for all account in the tenant. You need to be an Azure AD tenant admin to do this.
+
+> :warning: The next step requires you to go back to your msal-node-api registration.
+
+1. Now you need to leave the registration for **msal-react-spa** and *go back to your app registration* for **msal-node-api**.
+   - From the app's Overview page, select the Manifest section.
+   - Find the entry for `KnownClientApplications`, and add the Application (client) ID of the `msal-react-spa` application copied from the Azure portal. i.e. `KnownClientApplications: [ "your_client_id_for_TodoListSPA" ]`
+
+#### Configure the client app (msal-react-spa) to use your app registration
+
+Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
+
+> In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+1. Open the `SPA\src\authConfig.js` file.
+1. Find the string `Enter_the_Application_Id_Here` and replace the existing value with the application ID (clientId) of the **msal-react-spa** application copied from the Azure portal.
+1. Find the string `Enter_the_Tenant_Info_Here` and replace the existing value with your Azure AD tenant ID.
+1. Find the string `Enter_the_Web_Api_Scope_here` and replace the existing value with the *scope* you created earlier `api://{clientId-of-service}/access_as_user`.
+
+### Configure Security Groups (msal-react-spa)
+
+You have two different options available to you on how you can further configure your application(s) to receive the `groups` claim.
```

Comment:
>You [](http://example.com/codeflow?start=0&length=3)

add a note here : This sample utilizes a couple of groups in both the client app and the API. Note that it is not required to have group based authorization in both the client and the API. Depending on your requirement, you can choose to implement group based authorization in either the API or the SPA client and/or use a different set of security groups in both apps.